advanced/Scripts/webpage.sh

```diff
@@ -211,6 +211,10 @@ SetExcludeClients(){
 
 }
 
+Halt(){
+        nohup bash -c "sleep 5; halt" &> /dev/null </dev/null &
```

Comment:
Hadn't thought of that, and was trying to implement my change in a way that was as consistent as possible with what was there already.  But I agree that it might be better practice to use shutdown command (in both cases) with an appropriate message, in case this is running on a more general-purpose system and not a dedicated box.

Did you want to implement the change to both the Halt and Reboot functions?  Or should I do that?  (Keep in mind that I'm new to GitHub, and you could probably do this way faster than me!)